### DISCUSSION TITLE
Support for latest graphql library

### DISCUSSION THREAD

**Discussion by ts-ram** [yahoo/elide#2072] [Elide 5]
Are there plans to support newer version of the graphql library on which elide is based o to support extended scalar data types ?

**Comment by aklish** [comment]
We made an attempt to upgrade a few years ago, but nobody has asked for it and it was non-trivial.  

You can register custom scalars in Elide for GraphQL by creating a serde.  Here is the code that registers them:

https://github.com/yahoo/elide/blob/master/elide-graphql/src/main/java/com/yahoo/elide/graphql/GraphQLConversionUtils.java#L68-L79

We can definitely look at doing an upgrade if there is something that cannot be accomplished with the current version.

**Comment by aklish** [comment]
Elide 5.0.6 has the latest Graphql-java library.

**Reply by ts-ram** [reply]
Thanks @aklish
